pkg/api/v1/pod/util.go

```diff
@@ -194,7 +194,7 @@ func visitContainerConfigmapNames(container *v1.Container, visitor Visitor) bool
 }
 
 // GetContainerStatus extracts the status of container "name" from "statuses".
-// It also returns if "name" exists.
```

Comment:
the existing doc is more correct... it is describing what the boolean value represents

Reply:
done